### PR TITLE
Alex/dev

### PR DESCRIPTION
**Pull request by Alexdelia**
I made the `describe` binary

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,18 @@
-/target
+### datasets ###
+*.csv
+
```
Comment (justincapik) [inline .gitignore]:
```suggestion

```
we will use the `csv` in the git

File: src/describe/src/describe/compute.rs [hunk 2]
```diff
@@ -0,0 +1,404 @@
...
+			.collect::<Vec<f64>>();
+
+		arr.sort_by(|a, b| a.total_cmp(b));
+
+		let Some(first) = arr.first().copied() else {
+			return ret;
+		};
+
+		let mut min = first;
+		let mut max = first;
```
Comment (justincapik) [inline src/describe/src/describe/compute.rs]: get last for max (series is sorted)

File: src/describe/src/describe/compute.rs [hunk 3]
```diff
@@ -0,0 +1,404 @@
...
+			sum += value;
+		}
+
+		let mean = sum / arr.len() as f64;
+		let median = if arr.len() % 2 == 0 {
+			let mid = arr.len() / 2;
+			(arr[mid - 1] + arr[mid]) / 2.0
+		} else {
+			arr[arr.len() / 2]
+		};
```
Comment (justincapik) [inline src/describe/src/describe/compute.rs:108]:
```suggestion
		let median = arr[((arr.len() as float) / 2).floor()]
```

File: src/describe/src/describe/compute.rs (hunk 2 above)
Comment (Alexdelia) [reply src/describe/src/describe/compute.rs]: fixed in 1353d0d

File: src/describe/src/describe/compute.rs (hunk 3 above)
Comment (Alexdelia) [reply src/describe/src/describe/compute.rs:108]: `polars` `.median()` use the average if the len is odd

#### example:
- `median` of `[2, 4]` is `3`

**Review by justincapik — APPROVED** [review]
beautiful 👌

**Pull request merged**